src/ts/transformers/har.ts

```diff
@@ -134,15 +134,15 @@ export function transformPage(harData: Har, pageIndex: number = 0): WaterfallDat
     });
 
   const marks = Object.keys(pageTimings)
-    .filter((k) => (typeof pageTimings[k] === "number" && pageTimings[k] >= 0))
+    .filter((k: keyof PageTimings) => (typeof pageTimings[k] === "number" && pageTimings[k] >= 0))
```

Comment:
Hm good point. On the other hand this is only checked at TS compile time, so it just ensure that the types of the keys that do exist in the type-definition match correctly - which would automatically extend to all other un-typed keys with the same type at runtime.

The question is what is better - I'd perhaps just keep it for now.